dev-python/libcloud/libcloud-3.1.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="ssl(+)"
+
+inherit distutils-r1
+
+DESCRIPTION="Unified Interface to the Cloud - python support libs"
+HOMEPAGE="https://libcloud.apache.org/"
+SRC_URI="mirror://apache/${PN}/apache-${P}.tar.bz2"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples"
+
+RDEPEND="
+	>=dev-python/requests-2.5.0[${PYTHON_USEDEP}]
+
+"
+
+S="${WORKDIR}/apache-${P}"
+
+distutils_enable_tests pytest
+
+BDEPEND+="
```

Comment:
I think you should be OK to do this above RDEPEND because d_e_t will append it magically, but this isn't wrong

Reply:
Yeah I know I write this way as a reminder that distutils_enable_tests does stuff to BDEPEND :)